.aws/task-definition.json

```diff
@@ -25,37 +20,33 @@
       "logConfiguration": {
         "logDriver": "awslogs",
         "options": {
-          "awslogs-group": "awslogs-tis-template",
+          "awslogs-group": "awslogs-tis-revalidation-migration",
```

Comment:
Do we need the word "preprod" here?  In the terraform the name is like this "awslogs-preprod-tis-revalidation-migration"

Reply:
I have updated the name for prod and preprod. Thanks